yt/frontends/stream/data_structures.py

```diff
@@ -133,6 +157,36 @@ def __init__(self, ds, dataset_type=None):
     def _count_grids(self):
         self.num_grids = self.stream_handler.num_grids
 
+    def _icoords_to_fcoords(self, icoords, ires, axes=None):
+        """
+        We check here that we have cell_widths, and if we do, we will provide them.
+        """
+        if self.grid_cell_widths is None:
+            return super()._icoords_to_fcoords(icoords, ires, axes)
+        # Ideally this would be ported to something more efficient.  As it
+        # stands, this will use a pure python loop!
+        if axes is None:
+            axes = [0, 1, 2]
```

Comment:
If we use a tuple instead of a list, it'll be safe to use as a default value, an this would become unneeded

Reply:
I'm not totally sure I am ready to make this change, since using `None` is what's done in the baseclass.  But I will look into it.  This particular routine is also likely able to be made much more efficient with numpy tricks -- I just couldn't figure it out.